src/lib.rs

```diff
@@ -1520,6 +1512,8 @@ fn test_struct_size() {
 
 #[repr(transparent)]
 pub struct BpfInstruction(raw::bpf_insn);
+
+// Be aware of the Drop impl
```

Comment:
As per #261, please submit a PR to remove this, then rebase and we won't need this comment.

Reply:
unfortunately I can't use Handle for remove the `BpfProgram` `Drop` impl. I will see what I can do.